modules/RTC/TPCUtils.ts

```diff
@@ -299,6 +311,69 @@ export class TPCUtils {
         }
 
         return standardSimulcastEncodings;
+        // Dynamically determine number of simulcast layers based on resolution
```

Comment:
How is this code getting executed if it after the return statement? Am I missing anything here?